package.json

```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "build": "rollup -c",
     "examples": "webpack-dev-server --config webpack.config.js",
-    "lint": "eslint src/**/*.{js,jsx} --fix",
+    "lint": "eslint src/**/*.js --fix",
```

Comment:
Any reason to exclude `jsx` in particular?

Reply:
The repo doesn't have any jsx files. And it's not react based so won't have any.
And eslint throws and error with `npm run lint`
```
No files matching the pattern "src/**/*.jsx" were found.
```